js/coinflex.js

```diff
@@ -0,0 +1,2354 @@
+        //              }
+        //          ]
+        //     }
+        //
+        const transfers = this.safeValue (response, 'data', []);
+        return this.parseTransfers (transfers, currency, since, limit);
+    }
+
+    parseTransfer (transfer, currency = undefined) {
+        //
+        // fetchTransfers
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38432",
+        //         "toAccount": "38774",
+        //         "id": "758113170128764931",
+        //         "status": "COMPLETED",
+        //         "transferredAt": "1651428178967"
+        //      }
+        //
+        // transfer
+        //
+        //     {
+        //         "asset": "USDT",
+        //         "quantity": "5",
+        //         "fromAccount": "38422",
+        //         "toAccount": "38776",
+        //         "transferredAt": "1651704762775"
+        //     }
+        //
+        const currencyId = this.safeString (transfer, 'asset');
+        const timestamp = this.safeString (transfer, 'transferredAt');
+        const fromAccount = this.safeString (transfer, 'fromAccount');
+        const toAccount = this.safeString (transfer, 'toAccount');
+        let status = this.parseTransactionStatus (this.safeString (transfer, 'status'));
+        // if case of 'transfer', as status field is not available, 'transferredAt' prop should mean that transfer was done
+        if (status === undefined && timestamp !== undefined) {
+            status = 'ok';
+        }
+        return {
+            'id': this.safeString (transfer, 'id'),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'currency': this.safeCurrencyCode (currencyId, currency),
+            'amount': this.safeNumber (transfer, 'quantity'),
+            'fromAccount': fromAccount,
+            'toAccount': toAccount,
+            'status': status,
+            'info': transfer,
+        };
+    }
+
+    async buildOrderRequest (market, type, side, amount, price = undefined, params = {}) {
+        const clientOrderId = this.safeString (params, 'clientOrderId');
+        const maxCOI = '9223372036854775807';
+        if ((clientOrderId !== undefined) && Precise.stringGt (clientOrderId, maxCOI)) {
+            throw new InvalidOrder (this.id + ' createOrder() param clientOrderId should not exceed ' + maxCOI);
+        }
+        const orderType = this.convertOrderType (type);
+        // creating stop orders using type argument will mess up the unification logic (beacuse of missing market/limit). So, we have to use unified approach for sending stop orders
+        if (orderType === 'STOP') {
+            throw new ArgumentsRequired (this.id + ' createOrder() : to create a stop order, you need to specify the "stopPrice" param');
+        }
+        await this.loadMarkets ();
+        const order = {
+            'marketCode': market['id'],
```

Comment:
![image](https://user-images.githubusercontent.com/43336371/167108175-4e13d69b-0cd8-46f6-aa93-770fc6b5569c.png)
Despite the documentation saying the default `timeInForce` is GTC by experimentation I think it is IOC (see image attached) which is not very pleasant if the user is not aware. Should we add a `timeInForce: GTC` here?

Reply:
No, turns out you have entered an specific market https://coinflex.com/markets/LTC-USD where there are not sellers most of the time (empty orderbook for sellers side), so your `market` order didnt find match immediately and was cancelled. so, they are not converting orders to IOC, but it is the natural flag for `market` orders and cant be changed